components/docs-chef-io/content/automate/chef_infra_external_cookbooks_in_chef_automate.md

```diff
@@ -0,0 +1,35 @@
++++
```

Comment:
We need to add the link to this page from https://docs.chef.io/automate/infra_server/ 
@dishanktiwari2501  Please find the appropriate section to add it here.

Reply:
We can mention it here 
https://docs.chef.io/automate/infra_server/#set-up-the-chef-infra-server